### PR TITLE
build: move from stale GH app to stale action

### PR DESCRIPTION
**Pull request by MarshallOfSound**
As in title

Notes: none

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: 'Close stale issues'
+on:
+  schedule:
+    - cron: '30 1 * * *'
```
Comment (nornagon) [inline .github/workflows/stale.yml:4]:
```suggestion
    # 1:30am every day
    - cron: '30 1 * * *'
```

i hate crontab notation 😭

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6
```
Comment (nornagon) [inline .github/workflows/stale.yml:11]: pin to a hash pls

File: .github/workflows/stale.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        with:
+          days-before-stale: 90
+          days-before-close: 30
+          stale-issue-label: stale
+          stale-issue-message: >
+            This issue has been automatically marked as stale. **If this issue is still affecting you, please leave any comment** (for example, "bump"), and we'll keep it open. If you have any new additional information—in particular, if this is still reproducible in the [latest version of Electron](https://www.electronjs.org/releases/stable) or in the [beta](https://www.electronjs.org/releases/beta)—please include it with your comment!
+          close-issue-message: >
+            This issue has been closed as it was considered stale, this issue will not be monitored.  If this is a bug and you can reproduce this issue on a [supported version of Electron](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#timeline) please open a new issue and ensure a repro is provided.
+          exempt-issue-labels: discussion,security 🔒,enhancement :sparkles:
+          only-pr-labels: not-a-real-label
```
Comment (nornagon) [inline .github/workflows/stale.yml:21]: ?

File: .github/workflows/stale.yml [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    steps:
+      - uses: actions/stale@v6
+        with:
+          days-before-stale: 90
+          days-before-close: 30
+          stale-issue-label: stale
+          stale-issue-message: >
+            This issue has been automatically marked as stale. **If this issue is still affecting you, please leave any comment** (for example, "bump"), and we'll keep it open. If you have any new additional information—in particular, if this is still reproducible in the [latest version of Electron](https://www.electronjs.org/releases/stable) or in the [beta](https://www.electronjs.org/releases/beta)—please include it with your comment!
+          close-issue-message: >
+            This issue has been closed as it was considered stale, this issue will not be monitored.  If this is a bug and you can reproduce this issue on a [supported version of Electron](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#timeline) please open a new issue and ensure a repro is provided.
```
Comment (nornagon) [inline .github/workflows/stale.yml:19]:
```suggestion
            This issue has been closed due to inactivity, and will not be monitored.  If this is a bug and you can reproduce this issue on a [supported version of Electron](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#timeline) please open a new issue and include instructions for reproducing the issue.
```

**Review by nornagon — COMMENTED** [review]
🤔 i wonder if we should write a doc somewhere about "how to write a good bug"? incl. Fiddle instructions and so on.

File: .github/workflows/stale.yml (hunk 3 above)
Comment (MarshallOfSound) [reply .github/workflows/stale.yml:24]: Only way I can see for it to not close PRs as stale

File: .github/workflows/stale.yml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+        with:
+          days-before-stale: 90
+          days-before-close: 30
+          stale-issue-label: stale
+          operations-per-run: 1750
+          stale-issue-message: >
+            This issue has been automatically marked as stale. **If this issue is still affecting you, please leave any comment** (for example, "bump"), and we'll keep it open. If you have any new additional information—in particular, if this is still reproducible in the [latest version of Electron](https://www.electronjs.org/releases/stable) or in the [beta](https://www.electronjs.org/releases/beta)—please include it with your comment!
+          close-issue-message: >
+            This issue has been closed due to inactivity, and will not be monitored.  If this is a bug and you can reproduce this issue on a [supported version of Electron](https://www.electronjs.org/docs/latest/tutorial/electron-timelines#timeline) please open a new issue and include instructions for reproducing the issue.
+          exempt-issue-labels: discussion,security 🔒,enhancement :sparkles:
```
Comment (nornagon) [inline .github/workflows/stale.yml:24]: ugh this doesn't parse. you were totes right about unicode in yaml. wtf, this is 2022.
```suggestion
          exempt-issue-labels: "discussion,security \ud83d\udd12,enhancement :sparkles:"
```

**Review by nornagon — APPROVED** [review]

**Comment by nornagon** [comment]
yml file is maybe failing to parse but seems sus 🤨 so i think we should merge this and see if it works

**Pull request merged**